Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -1,71 +1,99 @@
-# ---------------------------------------------------------------------------- #
-#                               MERGE                                          #
-# ---------------------------------------------------------------------------- #
+import yaml
+vcfdir = "vcfDir"
+chromosomes = [f'Chr{n}' for n in range(1, 3)]
```

Comment:
Does this stay here or goes in the Snakemake (I know this is next step)

Reply:
this is for the snakefile  (I forgot to remove. done now). not sure about the import yaml so I kept here